build/Dockerfile-go1.20.6-solc0.8.13-ubuntu

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:20.04 as solc_0.8.13_builder
```

Comment:
Time to update with Ubuntu 22? (solc, go, and base)

Reply:
Let's consdier this later. Ubuntu 20.04 LTS is supported until April 2025, so we still have time